packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -204,25 +213,40 @@ class DropdownMenu extends StatefulWidget {
   /// The default width of the menu is set to the width of the text field.
   final MenuStyle? menuStyle;
 
+  /// Controls the text being edited or selected in the menu.
+  ///
+  /// If null, this widget will create its own [TextEditingController].
+  final TextEditingController? controller;
+
+  /// The value used to for an initial selection.
+  ///
+  /// Defaults to null.
+  final T? selectedValue;
```

Comment:
This should probably be called initialSelection to match other component properties, like initalDate, initialTime, initialPage, and to remind the developer that it's not a getter for the current selected value.

Reply:
Addressed. Thanks!